examples/mockups/product.html

```diff
@@ -0,0 +1,466 @@
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+            <div>
+              <a href="" class="ui bui button basic inverted fluid large">Using RTD as a technical writer</a>
+            </div>
+          </div>
+
+          <div class="column">
+            <div>
+              <h3 class="ui header"><<i>missing</i>> Illustration</h3>
+              <h3 class="ui header">Easy Documentation Deployment for Technical Teams</h3>
+              <p>
+                We deploy and host your team's documentation automatically. With Pull Request previews you can validate all your changes, before they go live. Once you commit, we keep your documentation up to date, across all versions of your software.
+              </p>
+              
+              <div class="ui relaxed divided list">
+
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Always up to date</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Automatic deploy</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Multiple Versions</div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+            <div>
+              <a href="" class="ui bui button basic inverted fluid large">How RTD works for teams</a>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </section>
+
+    <!-- Features -->
+    <section class="ui vertical segment light padded" id="features">
+      <div class="ui container">
+        <h2 class="ui header">Features</h2>
+
+        <div class="ui two stackable cards">
+
+          <div class="card">
+            <div class="content">
+              <h4>Webhooks</h4>
+              <p>
+                Whenever you push code to your favorite version control system,
+                whether that is Git, Mercurial, Bazaar, or Subversion, we will
+                automatically build your docs so your code and documentation are
+                never out of sync.
+              </p>
+            </div>
+            <div class="extra content">
+              <div class="ui labels">
+                <span class="ui label basic large"><i class="icon primary github"></i> GitHub</span>
+                <span class="ui label basic large"><i class="icon primary gitlab"></i> GitLab</span>
+                <span class="ui label basic large"><i class="icon primary bitbucket"></i> BitBucket</span>
```

Comment:
The card display works great here, and the icon labels are a nice touch to avoid making this such a prose heavy page -- it provides a bit to break up the content.

It's hard to make the labels not look like buttons though. Any opinions on not using `basic label` on these?

![image](https://user-images.githubusercontent.com/1140183/144313323-279171ca-e14b-4790-9565-6462029035f0.png)

That's `circular label`, `label`, and `inverted primary label`. I do like the `basic label` also though, it doesn't distract from the important parts of the content.

Reply:
> I do like the basic label also though, it doesn't distract from the important parts of the content.

Yes. I agree with you here. We can also explore other options but I don't mind that the labels look like buttons, as long as they are not distracting too much. I think we'll end up with more labels than the ones I used as an example and the multiple buttons per card will help removing the feeling of a call for an action. Also, maybe we can add some action: A tooltip explaining the label concept/tool?

Reply:
Yeah it's definitely a toss up for me, they all look like buttons in some way. +1 on using basic button for now, and we can perhaps try again once we know what each section list would consist of.

We will be leaning into the text here, there are plenty of options for each block that won't have a great corresponding icon.